client/src/main/java/io/pravega/client/stream/impl/Orderer.java

```diff
@@ -1,54 +0,0 @@
-/**
```

Comment:
We are going to want to keep this class if we intend to implement PDP-26 or PDP-28.